Core/Frameworks/Baikal/Core/Server.php

```diff
@@ -133,6 +133,8 @@ protected function initServer() {
 
         if ($this->authType === 'Basic') {
             $authBackend = new \Baikal\Core\PDOBasicAuth($this->pdo, $this->authRealm);
+        } elseif ($this->authType === 'Http') {
+            $authBackend = new \Sabre\DAV\Auth\Backend\Apache();
```

Comment:
I am not 100% sure. I have made tests only on Apache.
Reviewing Sabre code, I am not sure it is limited to Apache: it does use REMOTE_USER http server variable which doesn't seems to be limited to Apache HTTP server: https://secure.php.net/manual/en/reserved.variables.server.php